dbms/src/Storages/DeltaMerge/SSTFilesToBlockInputStream.cpp

```diff
@@ -64,22 +64,43 @@ SSTFilesToBlockInputStream::~SSTFilesToBlockInputStream() = default;
 
 void SSTFilesToBlockInputStream::readPrefix()
 {
+    LOG_INFO(log, "Read SST Files with MultiSSTReader {}", this->region->id());
+    std::vector<SSTView> ssts_default;
+    std::vector<SSTView> ssts_write;
+    std::vector<SSTView> ssts_lock;
+
+    auto make_inner_func = [&](const TiFlashRaftProxyHelper * proxy_helper, SSTView snap) {
+        return std::make_unique<MonoSSTReader>(proxy_helper, snap);
+    };
     for (UInt64 i = 0; i < snaps.len; ++i)
     {
         const auto & snapshot = snaps.views[i];
         switch (snapshot.type)
         {
         case ColumnFamilyType::Default:
-            default_cf_reader = std::make_unique<SSTReader>(proxy_helper, snapshot);
+            ssts_default.push_back(snapshot);
             break;
         case ColumnFamilyType::Write:
-            write_cf_reader = std::make_unique<SSTReader>(proxy_helper, snapshot);
+            ssts_write.push_back(snapshot);
             break;
         case ColumnFamilyType::Lock:
-            lock_cf_reader = std::make_unique<SSTReader>(proxy_helper, snapshot);
+            ssts_lock.push_back(snapshot);
             break;
         }
     }
+    if (ssts_default.size())
```

Comment:
boolean in `if(...)` statement is better
```suggestion
    if (!ssts_default.empty())
```